test/integration/plugins/root-hooks.spec.js

```diff
@@ -136,47 +136,38 @@ describe('root hooks', function () {
     });
 
     describe('support ESM via .js extension w/o type=module', function () {
-      describe('should fail due to ambiguous file type', function () {
-        const filename =
-          '../fixtures/plugins/root-hooks/root-hook-defs-esm-broken.fixture.js';
-        const noDetectModuleRegex = /SyntaxError: Unexpected token/;
-        const detectModuleRegex = /Cannot require\(\) ES Module/;
-
-        it('with --no-experimental-detect-module', function () {
```

Comment:
[Testing] Why remove the `--no-experimental-detect-module` tests? That's still something we'll have to support.